.github/workflows/build.yml

```diff
@@ -19,6 +19,16 @@ jobs:
         command: check
         use-cross: true
         args: --target ${{ env.TARGET }}
+    - uses: actions-rs/cargo@v1
+      with:
+        command: check
+        use-cross: true
+        args: --target ${{ env.TARGET }} --no-default-features
+    - uses: actions-rs/cargo@v1
+      with:
+        command: check
+        use-cross: true
+        args: --target ${{ env.TARGET }} --no-default-features --features framebuffer
```

Comment:
Two features are introduced but only one is checked?

Reply:
Basically: after this PR, there are three interesting cases to check: `[]`, `[framebuffer]`, and `[framebuffer, appctx`]. `default = [framebuffer, appctx]`, so that combination was already being tested by the existing check... this commit adds checks for the other two options.

I agree it's nonobvious, though. Maybe I should add a comment?